src/modules/mc_att_control/mc_att_control_params.c

```diff
@@ -114,6 +114,20 @@ PARAM_DEFINE_FLOAT(MC_ROLLRATE_D, 0.003f);
  */
 PARAM_DEFINE_FLOAT(MC_ROLLRATE_FF, 0.0f);
 
+/**
+ * Roll rate controller gain
+ *
+ * Global gain of the controller. Increase if the response is sluggish, decrease if
+ * it is overshoothing.
+ *
```

Comment:
Can you add something to explain the difference with MC_XXX_P gains? Something like:

```
This gains scales the P, I and D terms of the controller:
output = MC_XXX_K * (MC_XXX_P * error
                   + MC_XXX_D * error_derivative
                   + MC_XXX_I * error_integral)
Set MC_XXX_P=1 to implement a PID in the ideal form. 
Set MC_XXX_K=1 to implement a PID in the parallel form.